### PR TITLE
Added a function to save data for a number or years. Also added a place …

### PR DESCRIPTION
**Pull request by ReitzTimothy**
…to vizualize data over a year.

### CODE REVIEW

**Review by joshuaedwards0 — APPROVED** [review]

File: vizualize_data.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+
+def read_data(filename):
+    year_data = np.load(filename)
+    return year_data
+
+def graphyears(number_of_years,start_year,dates):
+    Months = ['January','March','May','July','September','November','January']
+    labels=[]
+    fig , ax = plt.subplots(figsize=(40,4))
```
Comment (joshuaedwards0) [inline vizualize_data.py]: As far as I can tell 'fig' is not being used. Same for the 'i' variable below.

File: vizualize_data.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    startDay = '01'
+    endDay = '01'
+    dateslist = []
+
+    startDate = str(startYear) + '-'+'01'+'-' + startDay
+    endDate = str(startYear+1) + '-' +'01'+'-'+ endDay
+
+    dateslist.append(np.arange(startDate , endDate , dtype='datetime64[D]'))
+    dateslist = np.array(dateslist)
+    return dateslist
```
Comment (joshuaedwards0) [inline vizualize_data.py:43]: Missing space between functions. My OCD cannot handle this.

File: vizualize_data.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    return dates
+
+def main():
+    number_of_years=input("Enter how many Years you want to graph: ")
+    start_year=input('Please enter a starting year: ')
+    #dates=make_dates()
+    dates = get_dates_for_graphing()
+    print(dates.shape)
+    graphyears(int(number_of_years),int(start_year),dates)
+if __name__ == '__main__':
```
Comment (joshuaedwards0) [inline vizualize_data.py:72]: Missing space.

**Review by joshuaedwards0 — CHANGES_REQUESTED** [review]

File: vizualize_data.py [hunk 4]
```diff
@@ -11,10 +11,11 @@ def read_data(filename):
     return year_data
 
 def graphyears(number_of_years,start_year,dates):
+
     Months = ['January','March','May','July','September','November','January']
     labels=[]
```
Comment (joshuaedwards0) [inline vizualize_data.py:16]: Just ran it and now it's throwing an error.  Apparently you're unpacking a tuple and that's what that variable is for. My bad I just saw that it wasn't being used :/

**Review by joshuaedwards0 — CHANGES_REQUESTED** [review]

**Review by eyneill777 — COMMENTED** [review]
Looks good to me, nice work man.  One of the refactoring tasks I was thinking about doing today is moving the functions for saving and loading data into their own separate file.  I realized that when you save the full data set into a numpy file it makes it too big to load the whole thing into memory and train on it so I am going to make it so you can split it into batches with their own files.

**Review by eyneill777 — APPROVED** [review]

**Review by joshuaedwards0 — APPROVED** [review]

**Pull request merged**